### ISSUE TITLE
Ask for login/password when setting a new OU or server url

### ISSUE THREAD

**Issue by adrianq** [EyeSeeTea/pictureapp#147]
We are going to use the SDK login system. Every time the user wants to change the OU or server url, user and password will be asked. However, push continue as current:  it won't be asked during the push process, and will use the current hard coded username/ password.

**Comment by adrianq** [comment]
Related to #148

**Comment by adrianq** [comment]
@ivanarrizabalaga As we discussed on our previous meeting there were a couple of unclear points. Let me try to shed some light on them:
-  When the user clicks either on the OU or on the url field, the login window should be displayed.
- The login window should display three fields: user, login and server url. As it is in the current sdk login window. The server url should be the current one, if any, although it should be editable.
- It would be good to remember the last user name (not the password) so the second he wants to change anything the username is filled by default together with the server url. Again this field should be always editable at any time.
- If the user provides a correct user/pw the field should become editable. Once he exits the preferences interface, these fields are not editable anymore and user/pw will be asked again if he comes back to the preference interface and tries to change OU or url.
- In order to retrieve the list of available OU we should use the hidden/internal user/pw. We only use the user credentials for allowing the user to change the fields but for the rest of the communication with the server (push, OU retrieving, etc) we have to use the internal/hidden one.

I think we have covered everything and this makes much more sense but, of course, if you think something is not clear just give me a shout
